examples/big_model_offloading/big_model_w8a8_calibrate.py

```diff
@@ -1,90 +0,0 @@
-import torch
-from datasets import load_dataset
```

Comment:
removed since this user flow will be too slow for use (e.g. issues that teknium ran into)